mttl/arguments.py

```diff
@@ -454,6 +460,35 @@ def __post_init__(self):
                 + "into account when computing `gradient_accumulation_steps`."
             )
 
+    def to_hf_training_args(self) -> "TrainingArguments":
+        from transformers import TrainingArguments
+
+        return TrainingArguments(
+            run_name=self.wandb_run_name or self.expert_name or self.finetune_task_name,
+            use_cpu=self.compute_strategy == "cpu",
+            overwrite_output_dir=True,
+            output_dir=self.output_dir,
+            per_device_train_batch_size=self.train_batch_size,
+            per_device_eval_batch_size=self.predict_batch_size,
```

Comment:
hm, `predict_batch_size` is not defined in this file. 

Sorry but idk now where`self.predict_batch_size` is it set now and why is it in `self`.

Reply:
predict_batch_size and train_batch_size is automatically injected by the DatasetConfig

Reply:
these automatic injections are confusion, idk. If we can do something about it here...